### PR TITLE
AnswerFilter mdl rm observerId add observerPhone

### LINKED ISSUE #258: Filter answers by phone number instead of id
Depends on https://github.com/code4romania/monitorizare-vot/issues/299

On the answers page, replace the `Observer Id` filtering option with `Observer Phone`

### PR DESCRIPTION
**Pull request by Armagedon0097**
- Added phoneObserver param to srvc

- Filter by observerPhone instead of observerId

Change makes the answers ready to be filtered by phone number. 
Not tested -> API needed for testing

Closes #258

### CODE REVIEW

**Comment by aniri** [comment]
looking good, thanks! will be merged after the API is ready and we can test it :)

**Comment by Armagedon0097** [comment]
Great, thanks.

File: frontend/src/app/services/answers.service.ts [hunk 1]
```diff
@@ -29,6 +29,7 @@ export class AnswersService {
 export interface AnswersPackFilter {
 	idNgo?: number;
 	idObserver?: number;
+	phoneObserver?: number;
```
Comment (georgebejan) [inline frontend/src/app/services/answers.service.ts:32]: isn't `observerPhone` the correct wording in english?

Comment (Armagedon0097) [reply frontend/src/app/services/answers.service.ts:32]: Yes indeed it is. Just went after the pattern from the prps above of putting the `what` before the `who`. 
Never mind :D

**Review by aniri — APPROVED** [review]

**Pull request merged**